### PR TITLE
Improved readability of KIP7 class comments and modified test code.

### PR DESCRIPTION
**Pull request by sirano11**
## Proposed changes
This PR has the following contents.
- Improved readability of KIP7 class comments.
- Modified KIP7 contract deploy test. 

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-java/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-java)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

refer #82 

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: core/src/main/java/com/klaytn/caver/tx/kct/KIP7.java [hunk 1]
```diff
@@ -658,27 +658,27 @@ protected KIP7(String contractAddress, Caver caver, TransactionManager transacti
     }
 
     /**
-     * Static method create KIP7 Standard Token handler connecting existed Contract
+     * Static method create KIP-7 Standard Token handler connecting existed Contract
      * The credential with the "KlayCredential Object" passed as a function parameter will control the contract
      * @param contractAddress KIP7 Standard Token Address
```
Comment (kjhman21) [inline core/src/main/java/com/klaytn/caver/tx/kct/KIP7.java:663]:
```suggestion
     * @param contractAddress KIP-7 Standard Token Address
```

Comment (kjhman21) [reply core/src/main/java/com/klaytn/caver/tx/kct/KIP7.java:663]: Let me know what do you think. There are more items to be changed like this. Do we need to change this? If yes, please do that for all other items in this code. If not, please ignore this comment.

Comment (sirano11) [reply core/src/main/java/com/klaytn/caver/tx/kct/KIP7.java:663]: This PR was intended to change the word "KIP7" written in the function description and function argument in the comments. So, I think it's a right to consistently change all items listed in function descriptions, function arguments. 
I will check and replace it once again.

Comment (sirano11) [reply core/src/main/java/com/klaytn/caver/tx/kct/KIP7.java]: I modified all item in function description and argument.

**Pull request merged**